packages/vue-renderer/src/renderer.js

```diff
@@ -150,6 +150,7 @@ export default class VueRenderer {
         if (isMFS) {
           // Cleanup MFS as soon as possible to save memory
           _fs.unlinkSync(fullPath)
+          delete this._assetsMapping
```

Comment:
Do we need asset mapping inside renderer for HMR?

Reply:
Yes, because client modern mode doesnt have modern renderer

Reply:
@clarkdo Would you please point to where we require it? We may find a way to keep deleting assetsMapping here.

Reply:
getPreloadFiles, sorry I’m on phone

Reply:
Any probelm here?

Reply:
It needs to be deleted since every rebuilding will lead to hash changed, so the mapping needs to be generated again 🤥

Reply:
It needs to be deleted since every rebuilding will lead to hash changed, so the mapping needs to be generated again 🤥